src/orchard/clojuredocs.clj

```diff
@@ -0,0 +1,35 @@
+(ns orchard.clojuredocs
+  "Find docs from ClojureDocs and retrieve the result as a map."
+  {:author "Masashi Iizuka"}
+  (:require
+   [clojure.edn :as edn]))
+
+(def cache (atom {}))
+
+(defn- map-from-key [f coll]
+  (reduce #(assoc %1 (f %2) %2) {} coll))
+
+(defn update-documents-cache!
+  "Load exported documents file from ClojureDocs, and store it as a cache.
+  A EDN format file is expected to the `export-edn-file` argument.
+  (e.g. https://clojuredocs-edn.netlify.com/export.edn)"
+  {:added "0.5.0"}
+  [export-edn-file]
+  (let [docs (-> export-edn-file slurp edn/read-string)]
+    (->> (:vars docs)
+         (group-by #(:ns %))
+         (reduce-kv #(assoc %1 %2 (map-from-key :name %3)) {})
+         (hash-map export-edn-file)
+         (reset! cache))
+    true))
+
+(defn find-document
```

Comment:
I wonder if we can't come up with a better name here, as the word `document` implies something else. Perhaps `find-doc` (for `documentation`). Can't think of a great name, but names are important.